src/cpp/session/SessionModuleContext.cpp

```diff
@@ -928,6 +928,11 @@ FilePath oldScopedScratchPath()
    else
       return userScratchPath();
 }
+
+Error rLibsUser(std::string* pRLibsUser)
+{
+   return r::exec::RFunction("Sys.getenv").addParam("x", "R_LIBS_USER").call(pRLibsUser);
```

Comment:
I didn't do that on purpose as it's possible an .Renviron script can modify that environment variable specifically.